DFC.ServiceTaxonomy.CompUi/Handlers/CacheHandler.cs

```diff
@@ -183,6 +194,22 @@ private async Task ProcessPage(Processing processing, ContentItem currentContent
         }
     }
 
+    private async Task ProcessJobProfilesLinkingtoSectorLandingPages(Processing processing)
+    {
+        var result = await _relatedContentItemIndexRepository.GetRelatedContentDataByContentItemId(processing);
+
+        if (result?.Count() > 0)
+        {
+            foreach (var item in result)
+            {
+                if (item.ContentType == nameof(ContentTypes.JobProfile))
+                {
+                    await _eventGridHandler.SendEventMessageAsync(item, ContentEventType.StaxUpdate);
+                }
+            }
+        }
+    }
```

Comment:
You know my preference for LINQ. There is nothing wrong with what you have now, so you decide if you want to update to something like the below.

```
result?
    .Where(x => x.ContentType == nameof(ContentTypes.JobProfile))
    .ToList().ForEach(x => _eventGridHandler.SendEventMessageAsync(x, ContentEventType.StaxUpdate));
```